sr_robot_launch/scripts/teach_mode_node

```diff
@@ -19,166 +19,28 @@ from controller_manager_msgs.srv import (ListControllers, LoadController,
-        """
-        success = True
-        hand_finder = HandFinder()
-        hand_quantity = len(hand_finder.get_hand_parameters().joint_prefix)
-        if hand_quantity == 2:
-            hand_robot_prefix = 'sr_bimanual_hands_robot/'
-        elif hand_quantity == 1:
-            hand_robot_prefix = 'sr_hand_robot/'
-        else:
-            raise ValueError('Hand Finder did not find a correct number of hands')
-
-        for hand_id in self.robot_ids:
-            change_control_type = rospy.ServiceProxy(
-                hand_robot_prefix + hand_id + '/change_control_type', ChangeControlType)
-            try:
-                resp1 = change_control_type(chng_type_msg)
-                if resp1.result.control_type != chng_type_msg.control_type:
-                    success = False
-            except (rospy.ServiceException, rospy.ROSException) as e:
-                rospy.logerr("Service call failed: %s" % (e,))
-                success = False
-
-        # Allow some time to reload parameters
-        rospy.sleep(self.time_to_reload_params)
-
-        if not success:
-            rospy.logwarn("Failed to change the control type.")
-
-        return success
-
-    def change_arm_teach_mode(self, teach_mode):
-        success = True
-#         set_teach_msg = SetTeachMode()
-#         set_teach_msg.teach_mode = teach_mode
-        for arm_id in self.robot_ids:
-            rospy.loginfo(
-                "Calling service %s", arm_id + '_sr_ur_controller/set_teach_mode')
-            change_teach_mode = rospy.ServiceProxy(
-                arm_id + '_sr_ur_robot_hw/set_teach_mode', SetTeachMode)
-            try:
-                resp = change_teach_mode(teach_mode)
-                success = resp.success
-            except rospy.ServiceException:
-                success = False
-        return success
-
-
 class TeachModeManager(object):
 
     def __init__(self):
+        robot_joint_names = ["ffj0", "ffj3", "ffj4",
+                             "mfj0", "mfj3", "mfj4",
+                             "rfj0", "rfj3", "rfj4",
+                             "lfj0", "lfj3", "lfj4", "lfj5",
+                             "thj1", "thj2", "thj3", "thj4", "thj5",
+                             "wrj1", "wrj2"]
         self.s = rospy.Service('teach_mode', RobotTeachMode, self.teach_mode)
-        self.ctrl_helpers = {"right_hand": ControllerHelper(["rh"], ["rh_"]),
-                             "left_hand": ControllerHelper(["lh"], ["lh_"]),
-                             "right_arm": ControllerHelper(["ra"], ["ra_"]),
-                             "left_arm": ControllerHelper(["la"], ["la_"])}
+        self.ctrl_helpers = {
+            "right_hand": ControllerHelper(["rh"], ["rh_"], robot_joint_names),
+            "left_hand": ControllerHelper(["lh"], ["lh_"], robot_joint_names),
+            "right_arm": ControllerHelper(["ra"], ["ra_"], robot_joint_names),
```

Comment:
arms do not have joints set in the robot_joint_names. How does that work?

Reply:
I don't know. The change I am making here is replacing local ControllerHelper with one in common_resources project which has robot_joint_names as a parameter instead of being hard coded in the contructor: https://github.com/shadow-robot/common_resources/blob/noetic-devel/sr_controllers_tools/src/sr_controllers_tools/sr_controller_helper.py#L34

Reply:
Ok, I approved since this is only migration PR, but there is something fishy going on here. Ill ask in slack.